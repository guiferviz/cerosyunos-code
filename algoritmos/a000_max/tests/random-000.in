28
81
572988686 1804639063 1480398526 225106804 1109897116 325412832 1296749939 28630304 1158059311 1757296480 2101434033 1147586604 1526275382 710964979 1008392040 20035288 83788874 1552864095 244058883 509982447 1531844034 1460454591 127178266 43108017 1461268428 526689029 1527485886 1121928368 494566116 1948286713 615108476 1202885440 1127589343 297566828 1837692992 20882666 375026702 1324374806 1704417724 564611273 918398029 333762455 1184242945 867555213 146410950 2033045042 1234856381 790750894 386461769 710947423 520978242 269419657 1111110188 2132404231 2095068388 1029656365 743650874 55704300 297369289 1597959322 546710575 254385133 1858338470 85560210 951211601 1816114097 1330156117 767823123 85627406 119388393 1900074372 2145229430 198081075 1019380719 1676965326 1603540941 347920684 1245016108 1324976552 1958026398 410288730
81
1843309119 135095962 1642222375 1517954347 1763553304 1181133981 203295854 819629765 1902875276 903053667 519743506 2078757766 721132689 77506166 689176768 52338502 1771833116 707081607 398759982 1350794298 1446730621 1693454649 552208353 884447669 824260891 93118451 1562205312 106262399 706976045 1290810199 658946991 592765072 2140561107 2132622061 1888539306 1128685396 720614174 1363737270 1893956345 1312968116 1053038563 1153002739 649616613 1542057149 380578501 1285879223 1349268890 238864702 1913858335 1593027633 1269433001 941926712 638418597 1968783364 1976277836 187746923 1540990444 1541768481 1255556862 942406268 1897676271 337966322 1416544910 1826418021 1520574828 796502368 226311476 1888578202 1693774430 1895804182 1724779685 1499918548 622244651 1792354964 1307828997 285401024 1454086709 302801396 1053427111 1474483383 546118934
81
48816065 498952174 1506782673 231516844 1486844827 685744604 821825351 1159656079 1146079497 1194309723 2073179259 1472828882 1341639941 1489456810 1618335011 343885417 234191706 2085076807 1208977624 371605645 231246763 359858122 241352018 1228214646 1190221725 470757354 1698015591 358588264 889517596 1596892623 1799254948 1693070247 962532391 1716208186 244115194 1367562483 1916017955 35314355 1258493456 2112859373 185590182 896398098 106262755 772540866 1589639697 1497943913 1628122433 1007427027 2115454800 2128483044 2124225962 1499738761 412938500 212197327 454974203 153717724 1638740359 757180534 2059751091 1062540263 247705827 1229698917 1046237891 444967822 940604060 1541060840 1494928688 1591226243 1510944887 809795767 1510414564 909216510 1300350404 1990470543 846855070 1483246390 1847990888 975323206 784078351 146393647 1717562540
81
580161549 1562693414 455633716 1602917031 358009488 800862888 931761841 1739903127 1238072841 425586409 1784926601 236706740 536729739 1878214319 1312559145 422837528 453594689 559394096 1720415702 1942000949 356224251 1778710684 1419480563 1014598002 1043641566 1138737651 2065625488 1644612500 443212166 669454452 1262239421 1473060466 1268239514 681259376 1536284532 2077059382 475137371 1340909012 2075808296 1381720333 2030039744 1480635684 359640436 1122098508 2109966442 1800806771 1075608036 1794658635 1683958443 1658426741 2144722077 1394257897 771795157 1429506636 141417392 2121152867 670780393 1205728824 1439881900 1257633286 2039774532 187146697 786234179 345651964 953699674 30595281 1499832327 272154946 1563974766 1140195775 1277173467 1724194296 539149271 1681075297 876786178 399513653 17963193 1854487890 1418956932 1641400876 1279354853
81
864297773 687838045 1694088519 807046729 1928624593 2013264670 2092143945 337852838 99798186 94819597 284267836 630986408 1055026037 983542683 253423101 1776400720 1350389094 1769416718 1600874476 570468457 2100275332 38339592 589884359 403065800 1828240015 933797159 1481048620 917062170 72365146 1793723264 808635680 996922242 1707664111 1436631864 1022179365 887084956 1530425477 367958157 1010477635 1241046358 1602744816 1816847734 1825088848 1613723511 443548934 19207266 158139391 1660852864 1379176055 947004764 1762932129 1913346569 327590732 469632970 582137755 450861592 370638727 472909138 1160890551 1819662425 735345330 998560607 570954490 1417742476 1224291883 2120335004 246001019 1582499787 374225851 1358970895 1553314849 877970607 695305033 434675602 948333037 717738007 1514850428 508114604 1293387389 1430361439 1775966198
81
346177806 159695962 1255364640 1736392919 1414911502 669694720 1678738396 1329592844 361828987 1371063052 1713362026 997563192 1019010337 1667294831 2093431296 994892400 1330019168 952300149 340038085 2085904231 1561772179 1630352886 1079720260 132877004 858909155 2052885412 1510490741 1486776315 157433700 1606769477 1769796743 1788834923 1500996779 639848516 1642790445 755850234 866968559 470496852 974054254 533335043 595209052 859175384 1847427636 2098766473 1332046768 724152906 231927882 1737095771 483406025 603661355 1684247214 1394924095 546437840 327738488 1296716190 1254774887 1830161140 237233863 1991911438 1693281566 660192980 245503529 1605191667 1731617394 1809739924 1028458849 436710960 1856509866 2097424015 546041486 640928660 1500378751 895666491 1001338141 710822295 2007395294 1738577715 1699942059 634541052 1989174984 1681725751
81
1417989984 1197927890 1192309016 1146225318 1267942563 126412712 1680692529 179338895 52229918 233616457 6737962 928036512 1164215511 2116456177 1359901706 185831505 204972186 949704459 1176816099 1266269898 1492612526 390443744 709864614 77498660 1676247560 1390229796 2009525593 941027990 1038608981 647932474 1482881652 1839844704 764209253 373073097 572418073 1768330402 4803624 1088189423 694695010 1166671715 1633712591 2135375554 209686842 1267893999 991712829 1328114861 729341827 1627241355 1914128880 1699667720 1291687000 746567782 405485936 1740869328 240045386 1306426238 1913718018 230116284 2063480588 1527813637 1560273134 1206273788 1943098112 337968799 573325047 289039968 1829077181 2095727776 589590200 1984194114 758618784 1762595691 398058223 702347348 580149430 1771046516 1101419445 2094551122 1625072367 2027948850 558710322
81
1030516157 50871321 250903283 900609536 1048653596 1470174374 674012053 869997011 1718625395 1663189223 2026365938 841060728 284198656 261504486 1235594016 200319809 197131936 972884588 398247116 2016090221 930484682 124665047 412688696 954664236 777111395 63051546 630489342 1950991236 1416373050 1454539753 1374858464 810651466 202790070 676306249 162070834 408080625 1946105096 939815030 499541466 1401593394 624841344 766523658 8792463 1863873667 671253938 974887071 218363414 1502352293 390258969 1644352200 1642985391 1070464223 1822460428 1202728307 47133046 1730909889 442626335 1390969800 721929762 639916444 699896500 906175711 846217520 1229326662 1903498754 905499833 1542941966 1035261282 239607060 829806222 563345009 1912317489 1489449990 278916113 1111117043 1000893627 765552535 1869094528 948931992 81024680 538946523
81
1121109532 1608079080 1225773395 1467411732 74846860 1105298609 1235716426 1258596349 875355610 209503763 1861405516 1413496083 482482361 684781906 13561781 1224397099 632262440 334446579 889772000 121332790 36884847 1496808406 732202631 484586355 1919313356 765597976 1758892463 2002399627 1346998993 769484157 932474114 662051572 1639908726 1088177943 2147348673 462101565 341706162 1466388230 1470880736 1494098352 2029910246 1047152285 648353719 232271162 740722481 1149269949 331084908 1332801232 1373482762 1544394462 913273202 707423746 1886911716 1431464759 1896105062 676350220 368838922 423508220 598246068 1174489169 815811277 1835393971 1396542183 22508976 1227121027 1694771040 2147281224 1391038820 1958211629 2017014590 935857826 864778883 2122443641 825702508 1345868526 606389138 395672340 506932668 382523125 927584820 1595884737
81
1056033423 1093398076 1761093637 1223299272 2014568010 203301382 1464821272 441853190 1239886756 689131013 224433217 868214392 1740305161 1260016211 547775576 1637350689 1781251803 1874949949 1454809669 1972333012 457750884 1563133485 1399372704 102886681 1657241445 766805157 1623848017 999893050 2118121168 1897364999 611174132 1717078171 1786669134 1163552239 1238112333 245746970 357168411 1685998308 1315544484 853529810 1763903111 1563110454 1947210686 382697297 687062132 2042814864 3937059 1719063081 937720614 915618312 681416374 1215781869 1491352420 1969647816 1859302356 1983441949 1951859584 1864587304 1723045758 1872779790 547052824 502217776 517951255 1861923591 243688116 333965124 1754027679 1582705409 749674533 413629836 1805224997 2053325594 1661935484 1151130470 1940982563 493111984 1198460524 875861988 355412665 152820446 1335084463
81
1147322073 1822585277 1969298074 79125392 350890486 532847105 1586785627 1882063608 389429323 739692836 1446449268 205256163 1191287320 1183878703 1784525684 215975936 178140509 405445114 1321028418 784579205 1565625883 1329320627 2081865186 2114741898 822565318 410629591 639257505 1993730826 459769704 387030710 515771133 1882288904 1034440022 1941006518 1192979356 1891748675 533196038 1269894621 2055555835 364975611 334967732 716721255 1689899218 300983887 194102966 1323003491 1436935659 1828803852 1198226968 2030113315 544050185 47835268 920758079 953402418 704863179 1289713708 1641765832 593163986 149970213 2065022300 664451848 2066585423 1943194521 1787807932 2049068428 641872504 1409862089 1924331032 1657203610 376414867 1102648700 1357308895 1357409569 304203702 1894422360 2102712855 1271598186 43711235 1077242400 2066231667 395888969
81
2104938013 2042649270 1133300488 43512838 1411066729 677644967 906748581 1928708454 1337966377 29745552 231699182 1681548322 1313967866 636698607 1589330270 911977479 492568896 850872319 317958214 482700259 624670786 2001221529 949097776 267653701 1497341101 1634822425 747780466 1812499350 1268101078 797714517 1930326822 1939388400 1638046724 602577622 1059771047 406037352 410782593 1045457813 1591015211 1211850896 265161757 1862243175 159286399 16386955 423526746 1222420963 90467651 834257207 869199909 2053829889 76251521 1510483332 731941955 995415935 1722549700 267282542 1565354020 264008341 441198257 593593091 587917138 1012209336 1805545586 1090916166 403349237 589403708 656355129 60953395 455275750 1324391010 1190168586 238121236 38332478 1538336691 1299770003 1848323418 1631355630 1729357639 1633188075 1275382118 1432313985
81
1596092800 262568844 1783202132 1468952487 48315548 1924686934 1448532549 116401110 447958641 633157470 469034666 675192486 170392961 996198904 812313019 1392371108 49671699 1151922385 1513661436 2046783566 1116254604 1333157987 1040420690 1989268758 1796111202 1387394933 1194502764 929366327 1927758772 605533221 756030856 1603375633 141937588 577470896 1614931836 1420825933 1847894372 2143647796 689323773 2076388689 1136538164 1442039603 1533208747 1054861736 1541322935 69437438 91034490 831538217 741625536 210836700 659123898 314751392 1954006330 646375422 139544204 1025642149 1316798862 1067647325 1459423356 735828618 1532658799 2011903398 207070351 216431583 492768003 1013087920 675350003 2039794327 1750012361 1123958128 720470909 1464255835 43952578 912421100 934629126 134416112 817544030 2036951109 1666739120 2121163629 667087324
81
687372578 1898396682 1005117428 1826389282 1344071469 1024996567 1246887231 2140761842 789224895 1782881348 2122437096 1915193878 674985255 170796863 646163772 1107612131 2117266750 178700481 897740578 61992291 1409685356 1112357232 1055162543 1373630128 112490541 31028176 969266258 1393337398 1533221313 1319287107 2002212968 665529667 1328336867 1124062066 1558983851 894113027 1853561196 1469788167 55843352 1679634601 1324363895 103489356 1801098981 1910881314 1474338480 172919622 1129411700 6871150 1199745466 840955908 1641572180 1980308166 1476664018 1507413524 562779613 301562123 2054893558 932341855 1479216004 65396777 1781402586 410841695 207560450 905140567 1784514351 1968127821 480055466 1050155852 1089064576 2119612707 1941504663 1949748418 2074120242 1415205993 1540695344 2036732843 40377751 38702018 855801221 40378907 920591941
81
587130585 2062751696 2070714236 1604507372 1262686512 1709975332 1434472855 1458985635 1536817514 1674903253 1149630090 1814332466 908277110 487642118 157572853 1879526163 277220243 309482634 2069402701 974231991 534640514 413821302 753214275 1333770095 1991741416 1464370525 1543484685 114666097 1123395 577201654 1959396645 251489808 623493178 1275680386 1171356806 1303755139 2087209530 1210199583 186204977 1854308636 1921855898 485895930 865829133 732204323 644142001 382632574 1032708179 309124996 1917267479 513274879 483361305 1475877290 1731347235 274002284 881467588 1616007110 292578979 1669570500 856970668 1518924864 1046870587 1816407977 863236940 1909767289 380094179 1438043738 1419214000 1704300600 496104608 749884078 1998413060 1721909128 1281888304 1907474491 1312618116 1081652375 321196755 1909237310 1497121362 1368430820 1441625186
81
505010450 101746869 772177751 948629829 365313369 744508160 81439567 1949823768 388448174 568919372 382083556 337176147 45655193 313292518 1784550716 1463737533 348518305 1526813278 547557149 1280383024 1073396251 1293269709 1643954889 17518579 1572006021 467567774 577933960 1408064712 1567963387 787146065 1874287974 844768226 1312348000 755102646 830571026 1671922523 462028047 491618051 1180235067 1560647879 1676114090 1115889877 1056321601 992952606 284357330 179452283 708976688 540357258 218447036 579037409 961386821 1791690881 374378552 1971841913 1975895303 808362917 1520253212 2137131451 2096324125 1545334414 725166077 347504627 1268105823 1110157882 701412855 604504617 1451981894 850958487 1187398752 1324279495 1777301278 1644066206 1237020190 1260583474 1007567110 1511808256 550245578 304816685 843505429 988513640 1157357085
81
1407230746 1437828879 1065824673 436072625 1422091103 1303127410 720296783 113553739 436561174 1088305844 1400565656 1320441241 1053277711 1785631181 328046975 1370966202 1515134317 530112601 1283590080 196119711 1151622684 93361795 1346320962 1552043189 541373180 1638691923 1150107801 959192454 1096006888 1687654185 674439828 18457034 228369848 529059659 190522990 824827292 2062876326 612446944 92311381 325740779 1825315268 614237992 1223462435 1108054074 1013998227 2010213328 1686357602 2105995143 1970719120 1838594435 857456330 853004183 316875747 653716246 1351716810 700512218 1760154650 1157502422 1522583980 1353410675 2023274603 2037757728 1415207837 1366891338 413712932 123878077 1328641149 1667821191 2122118708 565999495 1474812023 221848439 830117177 406268003 1368310013 747111883 1233939447 2141809515 1724732397 1657795031 931131988
81
712180563 305478103 59179087 1039473416 463070080 1482239617 941896975 448364244 1435967277 2047528305 1454026173 339581553 1855493349 522715268 1201636253 1371424456 1135759503 1240470334 1998131320 716673002 927420525 135975791 1252643498 1471310313 307168615 1665243376 581822679 880811909 351085217 1339873891 1933837429 566440132 1568619701 1499342872 947221584 1838108901 193948830 429081981 242405529 1779762587 561670908 1289903973 1045273873 412371296 1654850680 1365630618 325496584 270567997 1100742658 1272164693 954278042 1095187391 1809360898 982571725 986152765 1565297764 13374514 1446364892 1789539603 1698854282 1067850544 859839045 986905656 1064723671 2028500777 143669540 1304007860 219639109 699074287 1654517390 1883447031 494842587 2029581504 1756114344 1786050921 162740315 1273021465 1408942300 1208821420 616701878 552776258
81
572631179 266499572 1760375228 1473637368 1027109636 390045770 1547038444 431360165 1035009887 1607352408 995400885 159748669 740307204 428189667 637730390 308153528 1666239657 765671503 1968508441 2134954697 1517979216 51547524 952441567 2137255277 827854491 1778220067 217503032 368252748 1769346097 597012317 733556186 766612137 2110054160 500470412 732816450 1073856026 1434958484 392608879 33095341 1888026662 1966496996 1397992889 1962933421 1830854185 1100267042 979402851 1711266828 1419344831 2144619810 949242669 796046131 40085995 218915467 1458099406 81459105 1482183863 1104973115 823136074 179109538 706762412 1253683579 947353260 2046240402 1204652201 2071967412 1281740327 942446329 30181020 1068931006 1916323587 554452142 1344044059 510041104 1313416548 734409688 689713728 849545572 1182235221 2085535425 1131366041 1122242169
81
1909056099 269566743 756767606 507849424 362402359 949936411 1434277065 1546452958 1756645899 949322414 608086290 1835440222 427570779 699865610 412540015 1874852611 1007536648 577855234 1686738299 1278097854 1984931559 2128500464 1805749298 1160320376 786318876 1160035948 683678227 964782293 635130309 1597942853 1044758235 1731964425 138699970 1173403796 965497435 554835244 64912023 1042003582 1496744345 1388933669 1758546970 1665614602 1840702548 1263471464 577656663 1010348077 7630971 232186209 91840643 655743742 1721587138 1862175337 976369059 1656892078 163601037 891117882 1446641718 173630764 444256166 421175880 1617105270 1698787321 978942636 924610680 1590070646 572959795 1651143233 498546734 562873496 1092000302 1521029899 1544079822 1349560152 1179773807 16915381 809580505 48966905 615264515 784630762 44419401 1175098994
81
1807296541 1882762728 1064068462 2066081790 2091120551 658978703 1925778729 1041233096 194713148 1082629507 1808993059 323470481 57762316 1245389687 692194975 1928825537 163471043 392360928 1487836917 826261617 939006987 1464473821 226804168 583974393 2000013188 225392007 1252326357 1340699375 1645298560 1178850249 23111260 2136722642 1607474074 1318401032 1765311569 2122943640 1905903270 1243597557 86317449 1663614534 784657812 822502036 828442348 919891807 1675421751 564143124 856483869 1385859528 785601640 879491521 1472382193 163995544 1573940736 1277149028 1701809243 1063008507 554107748 2043299379 550961557 825957111 1715929511 547871864 1515159 1400836606 1779119802 1259397614 1987030775 1341893099 1125663592 1341484307 756634882 1127842782 831046714 1393520190 462939999 643175858 794450723 981421879 865298981 880912925 544478547
81
1337816440 1639748437 560934979 198131597 43602644 158672283 1451784720 1437672510 648185007 2062319832 1053412907 779992038 626327188 1442133480 1696770883 1226305386 1555478492 1999508877 2120052944 1946399228 646306810 1321973001 1755561097 79731140 478239279 1142061244 1809684100 2111393595 1340732000 208137825 2112734887 1894110544 933270377 505957503 648925434 974678546 1131637179 656924165 1050546440 1550159106 1601388448 1083252626 597515782 145248391 1430713857 622609635 2094913931 644221293 429895246 1259788018 1444825777 1133340259 69829543 405011024 1467814954 1703438952 1934892615 1604065546 1230134154 1224651506 1211808717 99516407 718459258 1506471709 729639693 761635820 986501874 1866087668 1631026766 653673040 697078452 1963057241 1054038194 2078027632 279681841 1825583907 392532882 1181085946 378004108 1048937130 257754252
81
640554934 1666478601 706249687 3702752 816048941 69623694 1750574226 1685610034 1282532174 1283061044 667197043 1366048043 1356345160 867344740 578821441 359500154 1593268248 898670438 1838435355 151669576 1108514294 222115818 615836290 1598951434 2041057385 1998608978 1487726201 301847308 929448705 1071343609 538299121 194463062 1433795116 1014602626 1700215616 968844991 518327183 1281882487 407084063 472163429 51089370 604973122 509488409 43527438 1223315252 560538953 980743803 1617889551 1266487787 814801196 246664299 520253671 1411280385 88425389 533928027 423877746 487688246 1200975956 1985312306 255197076 1396991721 5317235 578208716 771452222 1141660168 1293829801 1302337498 1408693390 565733912 342635677 1883793501 169410065 1416832627 856559801 521866297 1759097031 618626965 97366660 188787400 258407676 1302808235
81
116639765 1696136850 139429262 501757906 396652360 690046864 1727671244 627111254 1605122519 1104037238 402856151 908956448 614805394 739365970 917061226 880250453 78404478 2041568351 348674527 406205114 629893055 1233253509 513282332 864666696 273506399 118136921 388256671 1981262142 705641080 1098475951 791684374 942486723 329207004 1713565926 2121033946 1091294571 133834217 759522814 104647343 1771872070 24630529 1656607610 1740120594 1958386965 145446193 1176603878 1272993069 544019519 978512906 1615617508 668052779 875570754 1789449473 477070336 1587503806 189857109 991605659 2061716590 1935911693 469195364 1258315362 1345341719 1318892404 1009275729 1406964550 971761917 754343019 109754016 1836877206 339337130 529103051 16033594 746409813 1524270580 2095277888 1104744623 1718180972 1314455670 196850037 1618967400 2009702426
81
1404304785 1635118592 787764527 1084752650 369623059 77229598 1681012700 1717726675 1736278415 123733368 1611471772 1922470561 899559246 2134049910 1531032811 28184709 1187981419 785715936 98783105 914370974 785767722 716752031 500514291 1086261124 23997425 173509800 206129533 1951787892 1407189120 1601458167 632471179 1476725631 1134001227 33593927 2003451318 1093672271 1545508334 805289701 2070735723 1214872468 1747578564 1016540308 410631456 552774856 1027356473 602480261 1055327712 2035161607 1747325178 832033599 1677234842 2065116429 407190952 1155936244 1655711365 367037843 996572034 517244988 215082420 688493076 114978318 821990845 627514808 1580962086 233136105 1078070238 712151186 939882982 1011556151 909451646 254543751 1968366836 134808588 1479443556 1774109719 1600430267 1589315516 1344272529 1451029017 328914569 1871683326
81
1560927673 1595527369 240749145 451139050 55830798 1785993849 739602690 51475166 408346536 680389693 1072339054 589738441 1716646137 1566627531 141980563 1330717686 146212013 2020236200 1239599950 1202223721 60975686 132508044 1827627935 1918294707 675575009 949439469 367636589 22717276 565688579 514384664 131261367 405451081 139741417 2010347639 853368848 140427720 246672257 518845481 1804689534 1922933061 1592690429 388837041 247150769 725692740 1671404508 1862431160 614720047 1826527761 1379161907 230787579 354497797 932535448 179218032 86822085 1416081219 429389471 755042584 1379985837 1272854032 1645929020 1289007005 136218921 797565733 967820416 1039819476 568986184 1330395528 1114756606 62685297 1434475440 2011550746 1247277719 28906776 164659383 272665509 786823888 1235002957 663517485 409876770 355320762 812010183
81
1211442151 410747083 1185240 1592364701 2011144051 1840104829 1956540536 1214188014 804706862 1837663832 588564363 875317964 266490448 541828266 742035428 1125510817 1527991997 138325919 54856657 1279810950 789891192 515643948 1120992467 81322129 163239977 840823077 455073784 674696197 585720596 582615947 1551247129 839545411 1300177661 1985638607 1342412346 1150343583 109388457 278574764 837359935 926357051 572342535 670835920 1439278001 1160742534 282214539 249027465 966205833 832208650 1064830694 1916696408 1655729779 1509119417 776494169 923898452 261329911 800402874 842548457 2118571971 822960887 348547942 2040780088 523153411 164665117 1944856890 2104897052 1264151616 881152409 1276551348 793461748 1270747349 216787159 1866376179 886953602 1449725089 150486476 1577800442 1939421119 1176895304 1640131725 1683224186 1486698294
81
220559337 776874503 1224260341 513170298 1387212457 508658207 1656121182 185923095 2144802053 1098827857 17372189 844600246 1151769259 586147330 1814901080 604776668 39880615 183540870 1019836370 156566840 1159521759 458751615 1264626342 1145904046 1782551192 643003005 2077968773 1032648923 1097638399 1699262317 1557399808 1867183383 1587832848 518644891 1217632435 1798717587 1279842265 1147126427 1045372514 1147321810 429891497 1807915344 426694079 1528628616 1139365887 1441426398 435804066 68685124 1847435833 680180665 582013626 755104320 1072284782 981555990 1366817582 1378026628 1209045378 1563760709 1736840281 1953401008 292288594 1735888457 2144429066 1422811480 208286787 249303426 300527165 1410458368 1854284987 783652493 1051122678 561608450 782484 229407722 2011095316 399218443 1301179292 1300588226 1667775757 18753452 350283826
