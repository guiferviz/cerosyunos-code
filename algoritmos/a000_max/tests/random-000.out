2145229430
2140561107
2128483044
2144722077
2120335004
2098766473
2135375554
2026365938
2147348673
2118121168
2114741898
2104938013
2143647796
2140761842
2087209530
2137131451
2141809515
2047528305
2144619810
2128500464
2136722642
2120052944
2041057385
2121033946
2134049910
2020236200
2118571971
2144802053
